### ISSUE TITLE
Bump to latest Contour 1.19.x

### ISSUE THREAD

**Issue by xtreme-sameer-vohra** [knative-sandbox/net-contour#648]
Latest release at the moment is [Contour v1.19.1](https://github.com/projectcontour/contour/releases/tag/v1.19.1)

**Comment by carlisia** [comment]
Will do this today.

**Comment by carlisia** [comment]
Update: contour v1.18.3 is where the upgrade to k8s v0.22.0 libraries happened (see Commits on Aug 18, 2021 https://github.com/projectcontour/contour/compare/v1.18.3...main).

I tried upgrading contour + pinning the k8s version to net-contour's current v0.21.4 but it didn't work, because the client-go library was also upgraded to v0.22.0 and there was incompatibilities when running conformance tests.

And until https://github.com/knative/pkg/pull/2327 is merged, we can't upgrade to v0.22.0.

So we will release with the current contour v1.18.2 version. Closing this.

c/c @nak3
